azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/Pump.java

```diff
@@ -83,17 +87,19 @@ public Void call() throws Exception
     	PartitionPump capturedPump = this.pumpStates.get(partitionId);
     	if (capturedPump != null)
     	{
-			this.host.logWithHostAndPartition(Level.FINE, partitionId, "closing pump for reason " + reason.toString());
+            TRACE_LOGGER.info(LoggingUtils.withHostAndPartition(this.host.getHostName(), partitionId,
+                    "closing pump for reason " + reason.toString()));
 			retval = this.host.getExecutorService().submit(() -> capturedPump.shutdown(reason));
-    		
-    		this.host.logWithHostAndPartition(Level.FINE, partitionId, "removing pump");
+
+    		TRACE_LOGGER.info(LoggingUtils.withHostAndPartition(this.host.getHostName(), partitionId, "removing pump"));
     		this.pumpStates.remove(partitionId);
     	}
     	else
     	{
     		// PartitionManager main loop tries to remove pump for every partition that the host does not own, just to be sure.
     		// Not finding a pump for a partition is normal and expected most of the time.
-    		this.host.logWithHostAndPartition(Level.FINER, partitionId, "no pump found to remove for partition " + partitionId);
+    		TRACE_LOGGER.info(LoggingUtils.withHostAndPartition(this.host.getHostName(), partitionId,
```

Comment:
Suggest making this debug -- this is a spammy message that you're only going to care about when debugging in detail.